### PR TITLE
Be true to the Windows counterpart(Runs the engine on a terminal)

### PR DESCRIPTION
**Pull request by xplshn**

### CODE REVIEW

**Comment by AsciiWolf** [comment]
Sorry, but this is incorrect. The Quake 3 engine (all quake3-based games) start its own window with console log on Windows, not a terminal window. This however was never the case on Linux/Unix and all quake3-based games started without any console window. I would really like to keep this that way.

File: io.github.ec_.Quake3e.OpenArena.desktop [hunk 1]
```diff
@@ -3,7 +3,6 @@ Name=OpenArena (Quake3e)
 Comment=First person shooter
 Icon=io.github.ec_.Quake3e.OpenArena
 Exec=openarena-q3e
-Terminal=false
+Terminal=true
 Type=Application
 Categories=Game;ActionGame;
-PrefersNonDefaultGPU=true
```
Comment (AsciiWolf) [inline io.github.ec_.Quake3e.OpenArena.desktop:9]: Also, why did you remove this? This is an important variable for all laptop users that have two GPUs. Without this, the io.github.ec_.Quake3e.OpenArena Flatpak will always run on an integrated, low-end GPU instead of the proper discrete one.

Comment (xplshn) [reply io.github.ec_.Quake3e.OpenArena.desktop:9]: Arent you supposed to set your default GPU to your dedicated one?

Comment (AsciiWolf) [reply io.github.ec_.Quake3e.OpenArena.desktop:9]: Nope, default GPU is usually the integrated one since it has much lower power consumption. Dedicated one is used only for more demanding tasks, such as games.
